pkg/cmd/api/api.go

```diff
@@ -212,7 +218,7 @@ func apiRun(opts *ApiOptions) error {
 }
 
 func processResponse(resp *http.Response, opts *ApiOptions) (endCursor string, err error) {
-	if opts.ShowResponseHeaders {
+	if opts.ShowResponseHeaders && !opts.Silent {
```

Comment:
I don't think we need to change this conditional. In fact, if someone requested response headers, let's display them even in `--silent` mode. Maybe there is some use-case for that, and I think it's safe to allow because both `-i` and `--silent` flags are explicitly opt-in.

Reply:
Right. Makes sense. I will remove that.